tests/contrib/pytest/test_pytest.py

```diff
@@ -564,6 +564,80 @@ def test_will_fail():
         assert test_span.get_tag("error.msg") == "assert 2 == 1"
         assert test_span.get_tag("error.stack") is not None
 
+    def test_pytest_tests_with_internal_exceptions_get_test_status(self):
+        """Test that pytest sets a fail test status if it has an internal exception."""
+        py_file = self.testdir.makepyfile(
+            """
+        import pytest
+
+        # This is bad usage and results in a pytest internal exception
+        @pytest.mark.filterwarnings("ignore::pytest.ExceptionThatDoesNotExist")
+        def test_will_fail_internally():
+            assert 2 == 2
+        """
+        )
+        file_name = os.path.basename(py_file.strpath)
+        self.inline_run("--ddtrace", file_name)
+        spans = self.pop_spans()
+
+        assert len(spans) == 1
+        test_span = spans[0]
+        assert test_span.get_tag(test.STATUS) == test.Status.FAIL.value
```

Comment:
I am guessing no, since the other tests assert it, maybe an assert that exception tags don't exist?

(feel free to ignore if I am suggesting unnecessary work)

Reply:
we don't, because we have no handler *after* the exception has happened. It sucks a bit, but I think it's better than what we have. 
I'll add an assertion that exception tags don't exist 👍